data/state/accountsDB_test.go

```diff
@@ -1270,3 +1270,72 @@ func TestAccountsDB_MainTrieAutomaticallyMarksCodeUpdatesForEviction(t *testing.
 	hashesForEviction = ewl.Cache[string(rootHash2)]
 	assert.Equal(t, 3, len(hashesForEviction))
 }
+
+func TestAccountsDB_RemoveAccountSetsObsoleteHashes(t *testing.T) {
+	t.Parallel()
+
+	marshalizer := &mock.MarshalizerMock{}
+	hsh := mock.HasherMock{}
```

Comment:
add some empty spaces here so the test can be more readable

Reply:
Done.